contracts/0.8.25/vaults/Dashboard.sol

```diff
@@ -38,24 +38,24 @@ interface IWstETH is IERC20, IERC20Permit {
  * All these functions are only callable by the account with the DEFAULT_ADMIN_ROLE.
  * TODO: need to add recover methods for ERC20, probably in a separate contract
  */
-contract Dashboard is AccessControlEnumerable {
+contract Dashboard is Permissions {
     /// @notice Address of the implementation contract
     /// @dev Used to prevent initialization in the implementation
     address private immutable _SELF;
     /// @notice Total basis points for fee calculations; equals to 100%.
     uint256 internal constant TOTAL_BASIS_POINTS = 10000;
 
+    /// @notice Indicates whether the contract has been initialized
+    bool public initialized;
```

Comment:
let's move `initialized` closer to `vaultHub` var (right now it's between const and immutable)